common/determined_common/api/request.py

```diff
@@ -128,6 +130,21 @@ def get(
     )
 
 
+def stream(
```

Comment:
non-blocking: I think I'd slightly prefer to just expose the `stream` argument on `get`, but it's a small deal.